Author: Pino Toscano <toscano.pino@tiscali.it>
Description: Disable use of waitid() on Hurd
 The forkfd implementation with waitid uses flags not yet available on the
 Hurd; thus temporarly disable that, as the forkfd implementation works fine
 without too.
 .
 Most probably not upstreamable.
Forwarded: no (possibly not needed)
Last-Update: 2015-07-04

---
 src/3rdparty/forkfd/forkfd.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/src/3rdparty/forkfd/forkfd.c
+++ b/src/3rdparty/forkfd/forkfd.c
@@ -54,6 +54,9 @@
 #    define HAVE_PIPE2    1
 #  endif
 #endif
+#ifdef __GNU__
+#  undef HAVE_WAITID
+#endif
 
 #if _POSIX_VERSION-0 >= 200809L || _XOPEN_VERSION-0 >= 500
 #  define HAVE_WAITID   1
